react/OrderForm.tsx

```diff
@@ -117,9 +117,14 @@ export const OrderFormProvider: FC = ({ children }) => {
     const localOrderFormString = localStorage.getItem('orderform')
 
     if (localOrderFormString != null) {
-      const localOrderForm = JSON.parse(localOrderFormString)
-
-      if (localOrderForm.value !== UNSYNC_ORDER_FORM_VALUE) {
+      const localOrderForm = JSON.parse(localOrderFormString) as OrderForm
+
+      if (
+        localOrderForm.value !== UNSYNC_ORDER_FORM_VALUE &&
+        (!data ||
```

Comment:
Suggestion: Move the `if (loading || error || !data)` block above this one, so you don't need to check for `!data` here.